test/Microsoft.TestPlatform.AcceptanceTests/TestCaseFilterTests.cs

```diff
@@ -9,68 +9,72 @@ namespace Microsoft.TestPlatform.AcceptanceTests
     public class TestCaseFilterTests : AcceptanceTestBase
     {
         [CustomDataTestMethod]
-        [NETFullTargetFramework]
+        [NETFullTargetFramework(true, true)]
```

Comment:
Run only one test in each class in `inprocess`, Unless there is any behavior change.

Reply:
Fixed